### PR TITLE
Edit default group warnings/info

### PR DESCRIPTION
**Pull request by PanSpagetka**
https://projects.engineering.redhat.com/browse/RHCLOUD-4258

Screenshot from 2020-02-13 11-00-35
Screenshot from 2020-02-13 11-00-26

Screenshot from 2020-02-17 10-50-09
Screenshot from 2020-02-17 10-51-44

### CODE REVIEW

**Comment by mmenestr** [comment]
Hey @PanSpagetka looks great!! Modals look perfect, but have a few things for the pages:

1. Breadcrumbs are incorrect, but I think that is something that is being addressed with another story right?
2. The column header “Role name” should just say “Name”
3. The description is incorrect, it should be “These are the roles all users inherit by default. Adding or removing roles in this group will affect permissions for all users in your organization.”
4. The Role names don’t visually align with the Role header in the 3rd screenshot, but it does in the other screenshots, so I’m going to assume that you aligned it later on? Haha
5. Remove the “Edit group” button on the top right of the pages because the group name and description are not editable in this group!
6. The filter should say “Filter by role name” if possible

File: src/smart-components/group/role/add-group-roles.js [hunk 1]
```diff
@@ -46,12 +53,21 @@ const AddGroupRoles = ({
     return push(closeUrl);
   };
 
-  return (
-    <Modal
+  console.log(isDefault, isChanged);
```
Comment (karelhala) [inline src/smart-components/group/role/add-group-roles.js]: Forgotten console.log

**Comment by karelhala** [comment]
> Hey @PanSpagetka looks great!! Modals look perfect, but have a few things for the pages:
> 
> 1. Breadcrumbs are incorrect, but I think that is something that is being addressed with another story right?

Yeah, breadcrumbs are resolved in different issue.

> 2. The column header “Role name” should just say “Name”

That should be resolved in different PR, but since Robin is making changes here, let's update it as well.

> 3. The description is incorrect, it should be “These are the roles all users inherit by default. Adding or removing roles in this group will affect permissions for all users in your organization.”
> 4. The Role names don’t visually align with the Role header in the 3rd screenshot, but it does in the other screenshots, so I’m going to assume that you aligned it later on? Haha
> 5. Remove the “Edit group” button on the top right of the pages because the group name and description are not editable in this group!
> 6. The filter should say “Filter by role name” if possible

Same as with point 2.

> 
> Thank you!!

**Comment by PanSpagetka** [comment]
@mmenestr all points except second one should be fixed. Description is saved in database, it is just shown here. I am not sure how is default group created, but we should change the description during creation. I will check with backend guys on this.

**Pull request merged**